packages/amplify-cli/src/extensions/amplify-helpers/remove-env-from-cloud.js

```diff
@@ -28,6 +29,10 @@ async function removeEnvFromCloud(context, envName) {
     const notificationsModule = require(categoryPlugins.notifications);
     await notificationsModule.deletePinpointAppForEnv(context, envName);
   }
+  const { frontend } = context.amplify.getProjectConfig();
```

Comment:
I wouldn't remove codegen files on `env remove` -> Would remove it only when running `delete`